kafka: add production docs (#101)
kudobuilder/frameworks: repository/kafka/docs/latest/install.md

- Install the [KUDO controller](https://kudo.dev/docs/getting-started/)
- Install the [KUDO CLI](https://kudo.dev/docs/cli/)

> This guide explains the basic installation of the KUDO Zookeeper and KUDO Kafka. 

Reviewer: ```suggestion
This guide explains the basic installation for both KUDO Kafka and KUDO Zookeeper.
```
Let's mention Kafka first as these are the docs for this technology.

> To run the production-grade KUDO Kafka clusters please follow the running [KUDO Kafka in production](./production.md)

Reviewer: ```suggestion
To run a production-grade KUDO Kafka cluster, please read [KUDO Kafka in production](./production.md)
```
